ratbagd/ratbagd-device.c

```diff
@@ -195,7 +219,7 @@ static int ratbagd_device_commit(sd_bus_message *m,
 }
 
 static int
-ratbag_device_get_model(sd_bus *bus,
+ratbagd_device_get_model(sd_bus *bus,
```

Comment:
Pls do this in a separate commit. If you rebase, do `git reset HEAD^`, and only commit this line (click the blue bar left of the line in vscode to stage only one line).

Reply:
@K1ngjulien thanks for the separate commit, can you also adjust the indent in that commit? Looks like 2 spaces to much.

Reply:
should be fixed now